Makefile

```diff
@@ -0,0 +1,14 @@
+run-client:
+	go run cmd/client/main.go
+run-server:
```

Comment:
You should add lint stage before run and build commands to ensure code pass the linter checks.